electron/main.js

```diff
@@ -305,29 +311,21 @@ async function launchDaemonDev() {
 }
 
 async function launchNextApp() {
-  const nextApp = next({
-    dev: false,
-    dir: path.join(__dirname),
-    port: appConfig.ports.prod.next,
```

Comment:
just for context, port and env were removed as:
- next port is handled by the http server that handles the next requests
- env as was previously defined was invalid config